homeassistant/components/asuswrt/router.py

```diff
@@ -179,11 +180,46 @@ def __init__(self, hass: HomeAssistant, entry: ConfigEntry) -> None:
             self.hass, dict(self._entry.data), self._options
         )
 
+    def _migrate_entities_unique_id(self) -> None:
+        """Migrate router entities to new unique id format."""
+        _ENTITY_MIGRATION_ID = {
+            "sensor_connected_device": "Devices Connected",
+            "sensor_rx_bytes": "Download",
+            "sensor_tx_bytes": "Upload",
+            "sensor_rx_rates": "Download Speed",
+            "sensor_tx_rates": "Upload Speed",
+            "sensor_load_avg1": "Load Avg (1m)",
+            "sensor_load_avg5": "Load Avg (5m)",
+            "sensor_load_avg15": "Load Avg (15m)",
+            "2.4GHz": "2.4GHz Temperature",
+            "5.0GHz": "5GHz Temperature",
+            "CPU": "CPU Temperature",
```

Comment:
Can we standardize the style of the new IDs? I mean getting rid of the sensor prefix and slugify remaining e.g. 2.4GHz.

Reply:
I was waiting for this request 😁. The problem is this is mapping what is returned from the library, but I think we should `slugify` at bridge level and have the key properly formatted. I will try later how to do this.

Reply:
So maybe it's easiest to just slugify `description.key` in the sensor module? Or even slugify the entire `unique_id`:
```python
self._attr_unique_id = slugify(f"{router.unique_id}_{description.key}")
````

Reply:
This is done.